pkgs/tools/graphics/astc-encoder/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "astc-encoder";
+  version = "2.5";
+  outputs = [ "out" ];
+
+  src = fetchFromGitHub {
+    owner = "ARM-software";
+    repo = "astc-encoder";
+    rev = "${version}";
+    sha256 = "0ff5jh40w942dz7hmgvznmpa9yhr1j4i9qqj5wy6icm2jb9j4pak";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  # Built with CMake flags from https://github.com/ARM-software/astc-encoder/blob/main/Docs/Building.md
+  # NEEDHELP: set the correct flags for the target architecture (arm etc.)
```

Comment:
Probably needs to be something like https://github.com/NixOS/nixpkgs/blob/7248f89da2dab8bca3a9f4f11d0407d0c8e25e89/pkgs/development/libraries/libpostal/default.nix#L18.